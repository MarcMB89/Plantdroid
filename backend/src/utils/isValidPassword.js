function isValidPassword(password) {
  return password === this.password;
}
module.exports = { isValidPassword };
