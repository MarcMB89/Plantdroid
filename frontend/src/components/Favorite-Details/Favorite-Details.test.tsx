import React from 'react';
import {render} from '../../utils/test.utils';
import FavoriteDetails from './Favorite-Details';

let navigation: any = {
  navigate: jest.fn(),
};

describe('Given a FavoriteDetails Component', () => {
  describe('When it is rendered', () => {
    let screen: any;
    beforeEach(() => {
      screen = render(<FavoriteDetails navigation={navigation} />);
    });
    test('Then screen should match the snapshot', () => {
      expect(screen).toMatchSnapshot();
    });
  });
});


