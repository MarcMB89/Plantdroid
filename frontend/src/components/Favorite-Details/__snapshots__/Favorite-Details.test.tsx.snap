// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a FavoriteDetails Component When it is rendered Then screen should match the snapshot 1`] = `undefined`;
